Resources/Textures/_Nuclear14/Objects/Consumable/Food/Preserves/meta.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": 1,
+    "license": "CC-BY-NC-SA-3.0",
+    "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24",
```

Comment:
Replace this with your own copyright info. Such as "created by X for Nuclear14. Discord #"

Discord handle is optional and you can use anything you like to identify yourself such as GitHub username or nothing if you prefer.

The license is the actual copyright status for people reusing it which is fine. For ref https://creativecommons.org/share-your-work/cclicenses/